src/ui/components/Tour/Tour.tsx

```diff
@@ -60,125 +60,144 @@ const Tour: React.FC = () => {
-                  <>
-                    <div className={styles.h1}>
-                      One-click logs
-                      <svg className={styles.logPoint} viewBox="0 0 24 24" fill="currentColor">
-                        <path d="M0 0h24v24H0z" fill="none"></path>
-                        <path d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"></path>
-                      </svg>
-                    </div>
-                    <p>Now click the plus button on a line of code to set a print statement.</p>
-                  </>
-                )}
-
-                {!showConsoleNavigate && !showBreakpointAdd && showBreakpointEdit && (
-                  <>
-                    <div className={styles.h1}>One-click logs 🦄</div>
-                    <p>You can pass anything you want here, including objects and variables.</p>
-                    <p>Later we can even show you how to add a unicorn.</p>
-                    <p>Type something here and hit enter.</p>
-                  </>
-                )}
-
-                {hasCompletedTour && (
-                  <>
-                    <div className={styles.h1}>Check it out!</div>
-                    <p>Take a look at the console.</p>
-                    <p>
-                      Replay just re-ran your recording and retroactively added your print statement
-                      each time that line of code was called. 🤯
-                    </p>
-                    <p>And with that, you've graduated from the tour. Happy exploring!</p>
-                    <p className="mt-8">
-                      <a
-                        href="#"
-                        onClick={e => {
-                          e.stopPropagation();
-                          setShowConfetti(true);
-                          setShowPassport(true);
-                          setTimeout(() => {
-                            setShowConfetti(false);
-                            dismissTourNag();
-                          }, 2500);
-                        }}
-                        className="hover:cursor-hand whitespace-nowrap rounded-lg bg-white px-3 py-1 font-medium text-primaryAccent shadow-lg hover:bg-blue-50"
-                      >
-                        Thanks!
-                      </a>
-                      {showConfetti ? <Confetti /> : null}
-                    </p>
-                    <img
-                      src={`/images/passport/tour_grad-default.png`}
-                      className={styles.largeCompletedImage}
-                      style={{
-                        zIndex: 0,
-                        opacity: 0.75,
-                        bottom: `300px`,
-                        transform: `rotate(14deg)`,
-                      }}
-                    />
-                  </>
-                )}
-              </div>
-            )}
+            <div className={styles.intro}>
+              {isNewUser ? (
```

Comment:
IMO inline ternaries like this are kind of hard to read when they're multi-lined, especially when they're nested. I think it improves readability to break the subsections down into their own components, e.g.

```js
<div>
  {isNewUser ? <NewUserInstructions /> : <ReturningUserInstructions />}
</div>
```

This component isn't too large so this is more of a nit which you can ignore though. Just sharing a personal preference.

Reply:
Yeah, I was feeling that too. I'm going to merge this one and then move to this format in the next iteration, because things are gonna get more complex soon

Thank you!